python/ray/autoscaler/v2/tests/test_utils.py

```diff
@@ -520,11 +520,7 @@ def test_cluster_status_formatter():
 
 Node: fffffffffffffffffffffffffffffffffffffffffffffffffff00003
  Usage:
-  0.0/1.0 CPU
-
-Node Activity
---------------------------------------------------------
-(no activity)"""
+  0.0/1.0 CPU"""
```

Comment:
should we change this test to be consistent with the newly enforced format as well?

Reply:
It is, because there are no active nodes